www/src/components/navigation/navbar.astro

```diff
@@ -1,44 +1,64 @@
 import SidebarToggle from "./SidebarToggle";
+
+export interface Props {
+  mode?: "landing" | "default";
+}
+
 const currentPage = Astro.url.pathname;
+const { mode = "default" } = Astro.props;
+
+const navbarLinks: Array<{ href: string; label: string; newTab?: boolean }> = [
+  {
+    href: "/en/introduction",
+    label: "Docs",
+  },
+  {
+    href: "/en/faq",
+    label: "FAQ",
+  },
+  {
+    href: "https://github.com/t3-oss/create-t3-app",
+    label: "Github",
+    newTab: true,
+  },
+  {
+    href: "https://init.tips",
+    label: "T3 Stack",
+    newTab: true,
+  },
+];
 ---
 
 <nav
   class={currentPage === "/"
-    ? `flex flex-col relative justify-between items-center pt-3 z-50 text-slate-50`
-    : `flex flex-col relative justify-between items-center pt-3 bg-t3-purple-50 dark:bg-slate-900 z-50`}
+    ? `flex flex-col relative justify-between items-center pt-6 z-50 text-slate-50`
+    : `flex flex-col relative justify-between items-center pt-6 bg-t3-purple-50 dark:bg-slate-900 z-50`}
   aria-label="Global"
 >
   <div class="flex items-center justify-between w-full px-6 pb-2">
     <div class="flex justify-between items-center w-full md:w-auto">
       <div>
         <a class="" href="/">
-          <img src="/favicon.svg" width="60" height="60" alt="T3 Logo" />
+          <img src="/favicon.svg" width="45" height="45" alt="T3 Logo" />
         </a>
       </div><div class="hidden -space-x-1 md:flex md:ml-10">
-        <a
-          class="inline-flex items-center border font-medium relative text-base px-3 py-2 rounded-md border-transparent text-t3-purple-500 dark:text-t3-purple-100 hover:text-t3-purple-400"
-          href="/en/introduction"
-          ><span class="">Docs</span>
-        </a>
-        <a
-          class="inline-flex items-center border font-medium relative text-base px-3 py-2 rounded-md border-transparent text-t3-purple-500 dark:text-t3-purple-100 hover:text-t3-purple-400"
-          href="/en/faq"
-          ><span class="">FAQ</span>
-        </a><a
-          class="inline-flex items-center border font-medium relative text-base px-3 py-2 rounded-md border-transparent text-t3-purple-500 dark:text-t3-purple-100 hover:text-t3-purple-400"
-          href="https://github.com/t3-oss/create-t3-app"
-          target="_blank"
-          rel="noopener noreferrer"
-          ><span class="">Github</span>
-        </a><a
-          class="inline-flex items-center border font-medium relative text-base px-3 py-2 rounded-md border-transparent text-t3-purple-500 dark:text-t3-purple-100 hover:text-t3-purple-400"
-          href="https://init.tips"
-          target="_blank"
-          rel="noopener noreferrer"
-          ><span class="">T3 Stack</span>
-        </a>
+        {
+          navbarLinks.map((navbarLink) => (
+            <a
+              class={`inline-flex items-center border font-medium relative text-base px-3 py-2 rounded-md border-transparent ${
+                mode === "landing"
```

Comment:
Reviewing this on mobile so can't see the full change but why do we need this? If there is contrast issues, is it not better to adjust the bg on the docs-layout?

Reply:
I wanted to keep the color (at least for now) in the docs, whereas I tweaked it on the landing page to be much lighter. I didn't want to touch docs page for now so I added the `mode` prop

Reply:
Makes sense. There are other stuff needed on the docs page so it is perfectly fine for now :)